src/sales-channels/amazon/price-adjustment.md

```diff
@@ -25,6 +25,6 @@ _Intelligent Repricing Rule: Price Adjustment_
 
 |Field|Description|
 |---|---|
-|Price Action|Options:<br>**Decrease By**: Choose when you want the defined price source value to be decreased before listing to Amazon.<br>**Increase By**: Choose when you want the defined price source value to be increased before listing to Amazon.<br>**Match Competitor Price**: (Intelligent repricing rule only) Choose when you want to change your Amazon listing price to match the lowest competitor price, based on your competitor feedback and variance parameters. When selected, the **Apply** and **Adjustment Amount** fields are removed.|
+|Price Action|Options:<br>**Decrease By**: Choose when you want the defined price source value to be to be adjusted down, creating a lower price for the rule, before listing to Amazon.<br>**Increase By**: Choose when you want the defined price source value to be adjusted up, creating a higher price for the rule, before listing to Amazon.<br>**Match Competitor Price**: (Intelligent repricing rule only) Choose when you want to change your Amazon listing price to match the lowest competitor price, based on your competitor feedback and variance parameters. When set to `Match Competitor Price`, the **Apply** and **Adjustment Amount** fields are removed.|
```

Comment:
```suggestion
|Price Action|Options:<br/>**Decrease By**: Choose when you want the defined price source value to be to be adjusted down, creating a lower price for the rule, before listing to Amazon.<br/>**Increase By**: Choose when you want the defined price source value to be adjusted up, creating a higher price for the rule, before listing to Amazon.<br/>**Match Competitor Price**: (Intelligent repricing rule only) Choose when you want to change your Amazon listing price to match the lowest competitor price, based on your competitor feedback and variance parameters. When set to `Match Competitor Price`, the **Apply** and **Adjustment Amount** fields are removed.|
```

Reply:
I don't think this affects how it is rendered, but maybe consider to be consistent with other instances in this PR.

Reply:
thank you.  missed that.  don't recall in the output what happened without the /